source/agora/consensus/protocol/Nominator.d

```diff
@@ -707,6 +708,53 @@ extern(D):
         this.active_timers[type] = this.taskman.setTimer(
             timeout.msecs, { callCPPDelegate(callback); });
     }
+
+    /***************************************************************************
+
+        Used by the nomination protocol to randomize the order of messages
+        between nodes.
+
+        Params:
+            slot_idx = the slot index we're currently reaching consensus for.
+            prev = the previous data set for the provided slot index.
+            is_priority = the flag to check that this call is for priority.
+            round_num = the nomination round
+            node_id = the id of the node for which this computation is being made
+
+        Returns:
+            the 64-byte hash which is unsigned long long type
```

Comment:
```suggestion
            the 8-byte hash
```

It's 64-bit (8-byte), not 64-byte. And `unsigned long long` is C++ terminology, we should not use it here.